lib/rules/cli/4.1.yaml

```diff
@@ -322,6 +322,7 @@
   :cmd: oc explain
   :options:
     :resource: <value>
+    :api_version: --api-version=<value>
```

Comment:
@wangke19 can you please put them in alphabetical order?  Thx!

Reply:
@pruan-rht Updated, see latest commit. Thanks